### DISCUSSION TITLE
Thanks for the great home row mods post.

### DISCUSSION THREAD

**Discussion by evantravers** [precondition/precondition.github.io#13] [General]
Just finished my first scan through your epic home row mods post… and **well done**. It's just lovely, and I wish it had been available to me when I started playing with it.

**Comment by precondition** [comment]
It felt wrong that such a great concept was hidden in Reddit comments and Discord chat messages and I was surprised that no resource existed to make sense of all those confusing tap hold configuration settings so I became the change I wanted to see in the world.

Although some people now believe I'm the one who invented the concept. In a similar vein to Amerigo Vespucci who gave his name to the American continent even though he wasn't the one who discovered it but was the one to have written a popular book about the New World haha.
